packages/react/src/component-tests/IcSelect/IcSelect.cy.tsx

```diff
@@ -0,0 +1,2479 @@
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type(TYPE_ENTER);
+    cy.get("@icFocus").should(HAVE_BEEN_CALLED_ONCE);
+  });
+
+  //not working
+  it.skip("should call icBlur when it loses focus", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.get("ic-select").invoke("on", "icBlur", cy.stub().as("icBlur"));
+    cy.findShadowEl("ic-select", "button").focus();
+    cy.findShadowEl("ic-select", "button").should(HAVE_FOCUS).type(TYPE_ENTER);
+    cy.findShadowEl("ic-select", "button").focus();
+    cy.findShadowEl("ic-select", "button").should(HAVE_FOCUS).type(TYPE_ENTER);
+    cy.get("@icFocus").should(HAVE_BEEN_CALLED_ONCE);
+    cy.get("ic-select").blur();
+    cy.get("@icBlur").should("have.been.called");
+  });
+
+  it("should set the 'placeholder' class name if no option is selected", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.findShadowEl("ic-select", "button ic-typography").should(
+      HAVE_CLASS,
+      "value-text placeholder ic-typography-body hydrated"
+    );
+  });
+
+  // not sure
+  it.skip("should set the text colour to the primary text colour if an option is selected", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+        value="espresso"
+      />
+    );
+    // cy.checkHydrated("ic-select");
+    // cy.get("ic-select").shadow().find(IC_INPUT_CONTAINER).click()
+    // cy.get("ic-select").shadow().find(IC_MENU_LI).should('be.visible');
+    // .should("have.attr2, ARIA_SELECTED);
+  });
+
+  // not sure
+  it.skip("should change the text colour to the primary text colour when an option is selected", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+        value="espresso"
+      />
+    );
+  });
+
+  // not sure
+  it.skip("should add .menu-scroll to menu components when options height is over 320", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={coffeeOptions}
+        value="espresso"
+      />
+    );
+  });
+
+  it("should add .menu-scroll to menu components when options are initially set and then populated with large data set", () => {
+    mount(
+      <IcSelect
+        label="What is your favourite coffee?"
+        options={searchableCoffeeOption}
+        value="espresso"
+      />
+    );
+    cy.checkHydrated("ic-select");
+    cy.findShadowEl("ic-select", IC_INPUT_CONTAINER).type("Ko");
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
+    cy.findShadowEl("ic-select", "button.select-input").should(BE_VISIBLE);
+    cy.clickOnShadowEl("ic-select", IC_INPUT_CONTAINER);
```

Comment:
```suggestion
    cy.clickOnShadowEl("ic-select", "button.select-input");
```

Reply:
> 

actually that may not help - it's still failing for me

Reply:
i think that line can be removed completely, as i think the second click is closing the menu, which causes the test to fail.

It's also no clear to me what the purpose of typing "Ko" into the field is, in  the context of what this is testing

Reply:
Removed the ko line and the second click :)